terraform/production/main.tf

```diff
@@ -64,4 +64,44 @@ module "postgres_db_production" {
   multi_az = true //only true if production deployment
   publicly_accessible = false
   project_name = "platform apis"
-}
\ No newline at end of file
+}
+
+/*      DMS SET UP         */
+data "aws_ssm_parameter" "academy_username" {
+   name = "/academy/reporting-server/username"
+}
+data "aws_ssm_parameter" "academy_password" {
+   name = "/academy/reporting-server/password"
+}
+data "aws_ssm_parameter" "academy_hostname" {
+   name = "/academy/reporting-server/hostname"
+}
+module "dms_setup_staging" {
```

Comment:
Should this be production?

Reply:
I missed that, thanks for pointing it out